custom_components/drivvo/sensors.py

```diff
@@ -178,7 +174,7 @@ class DrivvoSensorEntityDescription(SensorEntityDescription):
         name="Refuelling Volume",
         icon="mdi:fuel",
         device_class=SensorDeviceClass.VOLUME,
-        state_class=SensorStateClass.MEASUREMENT,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
Fixes:
```
2025-03-03 00:42:33.060 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.xxxxxxxxx_volume_do_ultimo_abastecimento (<class 'custom_components.drivvo.sensor.DrivvoSensorEntity'>) is using state class 'measurement' which is impossible considering device class ('volume') it is using; expected None or one of 'total_increasing', 'total'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/hudsonbrendon/sensor.drivvo/issues
```